Vendor Types For XRFrame Resolving Incorrectly

The below was a work around to get around types, but the outputs do not work with the latest distributed versions:
 https://github.com/pmndrs/react-three-fiber/issues/2501
 
## Versions:
- `three`: 0.159.0
- `@types/three`: 0.159.0
- `three-stdlib`: 2.28.7
- `@react-three/fiber`: 8.15.11

 
## Related Code
 https://github.com/pmndrs/react-three-fiber/blob/master/packages/fiber/src/core/utils.ts
 
 ## Related Error
- node_modules/@react-three/fiber/dist/declarations/src/core/index.d.ts:14
- node_modules/@react-three/fiber/dist/declarations/src/core/loop.d.ts:32
- node_modules/@react-three/fiber/dist/declarations/src/core/store.d.ts:136
- node_modules/@react-three/fiber/dist/declarations/src/core/utils.d.ts:6

```
 error TS2694: Namespace '"C:/project/src/web/node_modules/@types/three/index"' has no exported member 'XRFrame'.

14 declare const invalidate: (state?: RootState | undefined, frames?: number) => void, advance: (timestamp: number, runGlobalEffects?: boolean, state?: RootState | undefined, frame?: THREE.XRFrame | undefined) => void;
                                                                                                                                                                                             ~~~~~~~

node_modules/@react-three/fiber/dist/declarations/src/core/loop.d.ts:32:117 - error TS2694: Namespace '""C:/project/src/web/node_modules/@types/three/index"' has no exported member 'XRFrame'.    

32     advance: (timestamp: number, runGlobalEffects?: boolean, state?: RootState | undefined, frame?: import("three").XRFrame | undefined) => void;
                                                                                                                       ~~~~~~~

node_modules/@react-three/fiber/dist/declarations/src/core/store.d.ts:136:225 - error TS2694: Namespace '""C:/project/src/web/node_modules/@types/three/index"' has no exported member 'XRFrame'.  

136 declare const createStore: (invalidate: (state?: RootState | undefined, frames?: number | undefined) => void, advance: (timestamp: number, runGlobalEffects?: boolean | undefined, state?: RootState | undefined, frame?: THREE.XRFrame | undefined) => void) => UseBoundStore<RootState>;
                                                                                                                                                                                                                                    
~~~~~~~

node_modules/@react-three/fiber/dist/declarations/src/core/utils.d.ts:6:41 - error TS2694: Namespace '""C:/project/src/web/node_modules/@types/three/index"' has no exported member 'XRFrame'.     

6 declare type _DeprecatedXRFrame = THREE.XRFrame;

```

 ## Possible Solution
 Remove type _DeprecatedXRFrame = THREE.XRFrame, and move to the XRFrame as part of https://www.npmjs.com/package/@types/webxr

--- a/packages/fiber/src/core/utils.ts
+++ b/packages/fiber/src/core/utils.ts
@@ -7,7 +7,7 @@ import { AttachType, catalogue, Instance, InstanceProps, LocalState } from './re
 import { Dpr, Renderer, RootState, Size } from './store'
 
 // < r141 shipped vendored types https://github.com/pmndrs/react-three-fiber/issues/2501
-// @ts-ignore
+/** @ts-ignore */
 type _DeprecatedXRFrame = THREE.XRFrame
 export type _XRFrame = THREE.WebGLRenderTargetOptions extends { samples?: number } ? XRFrame : _DeprecatedXRFrame
 

